CI/env.sh

```diff
@@ -2,47 +2,26 @@
 
 export jobs=`grep -c processor /proc/cpuinfo`
```

Comment:
Is this used?  And why override for GCC below?

Reply:
gcc compilation crashes if I used all the processors available....

did not dig much more

Reply:
it is used in our various `build_XXXX.sh` scripts

Reply:
Got it.  I guess I was sloppy in looking for where it was used by just searching in the PR on github and didn't think about the other files - oops.